README.md

```diff
@@ -58,6 +58,43 @@ subtle manual mistakes you can make when releasing from your terminal.
 
 You can release all the unpublished packages by running `release-plz release`.
 
+### Generate shell completions
+
+#### zsh
+To load completions in your current shell session:
+```sh
+$ autoload bashcompinit; bashcompinit
+$ source <(./release-plz generate-completions)
```

Comment:
```suggestion
$ source <(release-plz generate-completions)
```
you are assuming people have the `release-plz` binary in the local directory.
Isn't this better?

Reply:
Yes, indeed, that's my copy paste mistake! I did this in a hurry, so I didn't double check before my push! Sorry about that.